meshtastic/ble_interface.py

```
@@ -122,199 +153,259 @@ async def legacy_log_radio_handler(self, _, b):  # pylint: disable=C0116
+                    logger.debug("Subscribing to LEGACY_LOGRADIO_UUID")
+                    await self.client.start_notify(
+                        LEGACY_LOGRADIO_UUID, self.legacy_log_radio_handler
+                    )
+                if self.client.services.get_characteristic(LOGRADIO_UUID):
+                    logger.debug("Subscribing to LOGRADIO_UUID")
+                    await self.client.start_notify(LOGRADIO_UUID, self.log_radio_handler)
+                if self.client.services.get_characteristic(FROMNUM_UUID):
+                    logger.debug("Subscribing to FROMNUM_UUID")
+                    await self.client.start_notify(FROMNUM_UUID, self.from_num_handler)
             except Exception as e:
+                logger.error(f"Failed to start notifications: {e}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)